src/raftstore/store/worker/region.rs

```diff
@@ -48,6 +51,9 @@ const GENERATE_POOL_SIZE: usize = 2;
 
 // used to periodically check whether we should delete a stale peer's range in region runner
 pub const STALE_PEER_CHECK_INTERVAL: u64 = 10_000; // milliseconds
+                                                   // used to periodically check whether schedule pending applies in region runner
```

Comment:
Please format the comment.

Reply:
it is weird that `format` makes that way.